driver/bpf/fillers.h

```diff
@@ -2151,20 +2151,32 @@ static __always_inline bool bpf_kgid_has_mapping(struct user_namespace *targ, kg
 	return bpf_map_id_up(&targ->gid_map, __kgid_val(kgid)) != (gid_t) -1;
 }
 
-static __always_inline bool get_exe_writable(struct task_struct *task)
+static __always_inline struct inode *get_exe_inode(struct task_struct *task)
 {
 	struct file *exe_file;
 	struct mm_struct *mm;
+	
 	mm = _READ(task->mm);
 	exe_file = _READ(mm->exe_file);
-	if (!exe_file) {
-		return false;
+	if (!exe_file) 
+	{
+		return NULL;
 	}
 
 	struct inode *inode = _READ(exe_file->f_inode);
+
+	return inode;
```

Comment:
`NULL` should be checked outside otherwise the verifier could go crazy with some clang/kernel versions, so yes if the verifier allows us to do it, we can return like this `return _READ(exe_file->f_inode);`